packages/matchbox/src/components/Radio/Radio.tsx

```diff
@@ -53,7 +66,6 @@ const Radio = React.forwardRef(function Radio(props, userRef) {
           onFocus={onFocus}
           onBlur={onBlur}
           type="radio"
-          error={error}
           ref={userRef}
           {...describedBy}
           {...componentProps}
```

Comment:
Because we spread props, I think we'll need to include `React.ComponentPropsWithoutRef<'input'>`. And maybe omit `type` from it?